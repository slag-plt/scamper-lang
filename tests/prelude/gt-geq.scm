(> 4 5)
(> 5.0 4.99)
(> 0 0.1)
(> 7 9)
(>= 5.0 5)
(>= 5.1 5)
(>= 10 5)
(>= 0.02 0.03)
(>= 0.01 0.02)