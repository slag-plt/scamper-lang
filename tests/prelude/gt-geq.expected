#f
#t
#f
#f
#t
#t
#t
#f
#f
